app/javascript/controllers/dropzone_controller.js

```diff
@@ -31,13 +31,15 @@ export default class extends Controller {
     }
   }
 
-  checkForDuplicates(fileName) {
+  checkForDuplicates(file) {
+    const fileName = file.fullPath || file.name
     // Extract all filenames that are visible
-    const fileNames = this.fileNameTargets.map(target => { if (target.offsetParent !== null) return target.innerText.trim() })
+    const fileNames = this.fileNameTargets.map(target => {
+      if (target.offsetParent !== null) return target.innerText.trim().split(": ")[0]
```

Comment:
The `.split(": ")[0]` is a bug fix that is currently breaking duplicate filename checking.